src/app/[locale]/_component/feedbackMessage/feedbackMessage.tsx

```diff
@@ -67,8 +68,18 @@ export function FeedbackMessage({
             <Button
               onClick={
                 firstButton?.goBack
-                  ? () => pushBack()
-                  : () => pushWithLocale(firstButton.href || '/')
+                  ? () => {
+                      pushBack();
+                      if (firstButton.onClick) {
+                        firstButton.onClick();
+                      }
+                    }
+                  : () => {
+                      pushWithLocale(firstButton.href || '/');
+                      if (firstButton.onClick) {
+                        firstButton.onClick();
+                      }
+                    }
```

Comment:
How is this related with this PR?

Reply:
Yes, I introduced it to have a callback for those buttons in the component to handle the call back events, as each page is different